engine-src/Game/LambdaHack/Client/UI/HandleHelperM.hs

```diff
@@ -305,11 +301,12 @@ skillsOverlay aid = do
       skillValue = EM.singleton monoFont $ offsetOverlayX skValue
   return (EM.unionsWith (++) [skillLab, skillDescr, skillValue], kxs)
 
+-- extract a set of places for each level, counting number of each kind of PlaceEntry
 placesFromState :: ContentData PK.PlaceKind -> ClientOptions -> State
                 -> EM.EnumMap (ContentId PK.PlaceKind)
                               (ES.EnumSet LevelId, Int, Int, Int)
 placesFromState coplace ClientOptions{sexposePlaces} s =
-  let addEntries (!es1, !ne1, !na1, !nd1) (!es2, !ne2, !na2, !nd2) =
+  let addEntries (!es1, !ne1, !na1, !nd1) (!es2, !ne2, !na2, !nd2) =  -- enum set, number entries, number arounds, number exists (dunno why it starts with d)
```

Comment:
Correct. I use 'd' instead of 'e' for exists, because 'e' is already taken (for 'e'ntries) and 'd' is the next letter in alphabet. :)